features/support/publishing_cuke_helpers.rb

```diff
@@ -26,6 +26,8 @@ def make_post(text)
   def submit_publisher
     txt = find("#publisher #status_message_text").value
     find("#publisher .btn-primary").click
+    # wait for the publisher to be closed
+    find("#publisher")["class"].should include("closed")
```

Comment:
Interesting, this now [fails](https://travis-ci.org/diaspora/diaspora/jobs/213826386#L2763) as `expected "mention_popup publisher row" to include "closed"`, what's the link with "mention_popup" here... ? Is that the infobulle which is open with when getting_started is on?